frontend/scripts/react-components/profiles/place-summary.component.jsx

```diff
@@ -52,9 +52,9 @@ class PlaceSummary extends React.PureComponent {
         onValueSelected: onYearChange
       }
     ];
-    const soyValue = soyArea !== null && soyArea !== 'NaN' ? formatValue(soyArea, 'area') : '-';
```

Comment:
@sorodrigo the NaN bit was copy-pasted from this

Reply:
Yeah it’s out of the scope of this PR. Maybe just add a ticket to PT?